geomesa-convert/geomesa-convert-avro-schema-registry/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>geomesa-convert_2.11</artifactId>
+        <groupId>org.locationtech.geomesa</groupId>
+        <version>2.3.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>geomesa-convert-avro-schema-registry_2.11</artifactId>
+    <name>GeoMesa Convert Avro Schema Registry</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-convert-avro_${scala.binary.version}</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.avro</groupId>
+                    <artifactId>avro</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.avro</groupId>
+            <artifactId>avro</artifactId>
+            <version>1.8.2</version>
```

Comment:
is 1.8.2 required?

Reply:
if it can work with 1.7.7 it might be nice to have a property avro.version and you can override it in your custom build

Reply:
well in your packaging i mean.

Reply:
I don't believe it works with 1.7

Reply:
we could add a `avro.confluent.version` property - it's used here and in the confluent KDS impl